infrastructure/aat.tfvars

```diff
@@ -11,3 +11,5 @@ capacity = "2"
 feature_paid_in_full =  "true"
 feature_admissions = "true"
 feature_testing_support = "true"
+feature_mediation = "true"
```

Comment:
`feature_mediation_pilot = "true"` ??? we have this enabled in demo

Reply:
done